### PR TITLE
update to version 2.0.0

### LINKED ISSUE #10: Problem: out of date library
Please update the library since the used endpoints are deprecated and some features like staking are missing.

### PR DESCRIPTION
**Pull request by linfeng-crypto**
- support many types of message in transaction
- require Python >= 3.7
- add many amino types

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,3 +6,15 @@ This log documents all public API breaking backwards incompatible changes.
 
 - Added
   - Code for address generation and transaction signing
+
+## 2.0.0
+
+- Added
```
Comment (tomtau) [inline CHANGELOG.md:12]: this doesn't say what was added -- should this be a header instead of a bullet point?

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,3 +6,15 @@ This log documents all public API breaking backwards incompatible changes.
...
+
+- add `timeout_height` argument in transaction
+
+- add amoni types including `Coin`, `StdFee`, `StdSignDoc` and many transaction message
```
Comment (tomtau) [inline CHANGELOG.md]:
```suggestion
- add amino types including `Coin`, `StdFee`, `StdSignDoc` and many transaction messages
```

**Review by tomtau — COMMENTED** [review]
looks all right -- I guess direct protobuf signing can be added in some future version?

**Review by leejw51crypto — APPROVED** [review]
lgtm

**Pull request merged**